src/d_pwr_led.py

```diff
@@ -0,0 +1,92 @@
+import os
+import time
+
+
+
+import time
```

Comment:
muss das zweite Mal unbedingt sein?

Reply:
was meinst Du damit? Das erste mal habe ich tim an sich impoertiert beim zweiten mal explizit sleep. Ich hätte auch sleep mit time.sleep ansprechen können. Habe ich in dem Fall nicht gemacht, weil sleep in vielen Sprachen geläufig ist und es im code eher stört wenn da noch ein "time." davor steht